src/components/atoms/InputRange/InputRange.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ChangeEvent, useState } from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import InputRange from './InputRange';
+import { axe } from 'jest-axe';
+
+describe('<InputRange />', () => {
+  it('renders and updates as expected', async () => {
+    const ControlledInputRange = () => {
+      const [value, setValue] = useState(75);
+      const onChangeHandler = (event: ChangeEvent<HTMLInputElement>) => setValue(Number(event.target.value));
+
+      return <InputRange value={value} onChange={onChangeHandler} />;
+    };
+
+    render(<ControlledInputRange />);
+
+    const input = (await screen.findByRole('slider')) as HTMLInputElement;
+
+    expect(input.value).toEqual('75');
+
+    fireEvent.change(input, { target: { value: '25' } });
+
+    expect(input.value).toEqual('25');
+  });
+
+  it('renders the component with no a11y violations', async () => {
+    const { container } = render(<InputRange aria-label="slider" defaultValue="51" />);
+    const results = await axe(container.innerHTML);
+    expect(container.firstChild).toMatchSnapshot();
+    expect(results).toHaveNoViolations();
+  });
```

Comment:
I wonder what happens if someone does:
```jsx
<InputRange defaultValue="50" min="10" max="40" />
```
and whether we should throw an error accordingly? 🤔

Reply:
it's just a "wrapper" of a regular <input type="range" /> (https://developer.mozilla.org/es/docs/Web/HTML/Elemento/input/range).

We are not doing any magic there. So if the user puts this values, We will show a range with min=10, max=40 and value=40 (in this use case).